src/ruby/spec/basic_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe SymEngine do
+    before :each do
+    end
+
+    describe SymEngine::Basic do
+        before :each do
+        end
+
+        describe '.new' do
+            context 'with no arguments' do
+                it 'returns a Basic object' do
+                    expect(SymEngine::Basic.new).to be_an_instance_of SymEngine::Basic
+                end
+            end
+        end
+    end
+end
```

Comment:
For now, we are trying to make all the tests pass. The operator methods will only pass when Basic has been initialized with some value, and I haven't written any initialisation methods. The operator methods show an error now.